core-feature-pack/legacy-feature-pack/pom.xml

```diff
@@ -0,0 +1,408 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2010, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly.core</groupId>
+        <artifactId>wildfly-core-feature-pack-parent</artifactId>
+        <version>13.0.0.Beta1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>wildfly-core-feature-pack</artifactId>
+
+    <name>WildFly: Core Feature Pack</name>
+    <packaging>pom</packaging>
+
+    <properties>
+        <license.directory>${project.build.directory}/${project.build.finalName}/content/docs/licenses</license.directory>
+        <common.resources.directory>${basedir}/../common/src/main/resources</common.resources.directory>
+        <ee-8-api.resources.directory>${basedir}/../ee-8-api/src/main/resources</ee-8-api.resources.directory>
+        <common.license.directory>${basedir}/../common/src/license</common.license.directory>
+        <ee-8-api.license.directory>${basedir}/../ee-8-api/src/license</ee-8-api.license.directory>
+    </properties>
+
+    <dependencies>
+
+        <!-- module and copy artifact dependencies -->
+
+        <!-- Bring in all the common deps in one go -->
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>wildfly-core-feature-pack-common</artifactId>
+            <type>pom</type>
+        </dependency>
```

Comment:
related to my comment above, you don't exclude `Jakarta.json:jakarta.json-api` as you did in the Galleon feature pack. Is it intended?

Reply:
It looks like it was not needed in the Galleon one so I removed it.